src/ActionModal/utils/cosmos-signing.js

```diff
@@ -0,0 +1,121 @@
+// adds the signature object to the tx
```

Comment:
I extracted a bunch of the functions from the `cosmos-api` library. Can instead export the needed functions from there.

Reply:
I'm a bit confused here. I thought we were going to create this in the extension? 

Also, are we going then to use my approach in #3874 to create a lunieTransaction (TransactionV2) or is that not needed having this?

Wouldn't like to write too much code that won't be later used 😓